src/frontend/identity/index.ts

```diff
@@ -31,6 +60,17 @@ export class Identity extends KojiBridge {
     return hasGrants;
   }
 
+  /**
+   * Requests the specified authorization grants from the user for the Koji.
+   *
+   * @param   grants           List of {@link AuthGrantCapability| authorization grants} to request from the user.
```

Comment:
I'm wondering if we should do something like this instead? 

`@param {AuthGrantCapability[]} grants List of grants to check for the user.`

Reply:
Yeah, I was wondering whether we needed the link at all. The HTML doc does the linking automatically already. If the JSON can be parsed in the same way, we probably don't need the link or the explicit parameter type (since it's in the Typescript)